### DISCUSSION TITLE
Do you offer provisioning of kyma clusters ?

### DISCUSSION THREAD

**Discussion by Thilaknath** [SAP/terraform-provider-btp#454] [Q&A] [answered]
I am more interested in learning if its possible to provision kyma clusters targeting the offered hyperscalers. Kindly let know.

**Comment by lechnerc77** [comment, marked as answer]
@Thilaknath Yes this is possible. You can use the `subaccount_environment_instance` for that. Here is the link to the [documentation](https://registry.terraform.io/providers/SAP/btp/latest/docs/resources/subaccount_environment_instance) which also contains a example for Kyma. You also find some information about the provisioning in our [samples repository](https://github.com/SAP-samples/btp-terraform-samples/tree/main/in-development/modules/envinstance-kyma). 

Be aware that the management of Kubernetes resources must be managed via the [Kubernetes provider](https://registry.terraform.io/providers/hashicorp/kubernetes/latest).
